storage/memory/memory.go

```diff
@@ -270,11 +270,8 @@ func (s *MemoryBackend) Write(ctx context.Context, store string, deletes storage
 
 	now := timestamppb.Now()
 
-	if err := validateTuples(s.tuples[store], deletes, writes); err != nil {
-		return err
-	}
-
 	var tuples []*openfgapb.Tuple
+
 Delete:
```

Comment:
Let's refactor to drop these control flow labels. They are confusing and this logic can easily be rewritten to avoid them.